### ISSUE TITLE
Vendor Dashboard: Unable to navigate to other settings menu if the Settings WP Endpoint is translated.

### ISSUE THREAD

**Issue by tanjiralmamun** [weDevsOfficial/dokan-wpml#12]
**Description of the Bug:**
Unable to navigate to other settings menu from the secondary language if the **_Settings WP Endpoint_** is translated.

If trying to visit the settings menu, it stays on the **_Settings > Store_** panel and doesn't redirect it into the settings menu.

**_Video URL:_** http://somup.com/crfhIEbJIO

**How to reproduce:**
- Translate the **_Vendor Dashboard > Settings_** WP Endpoint
- Visit the Vendor Dashboard
- Click on the Settings menu from the secondary language.
- It stays on the **_Settings > Store_** and doesn't redirect it into the settings menu.

**Expected Behavior:**
The Settings menu should be navigatable properly.

**System Status:**
Dokan Lite: 3.2.4
Dokan Pro: 3.2.2
Dokan WPML: 1.0.3
WordPress: 5.7
WooCommerce: 5.1.0
WPML Multilingual CMS: 4.4.10

**Comment by strategio** [comment]
@tanjiralmamun, I created 2 PR:
- In Dokan Lite (https://github.com/OnTheGoSystems/dokan/pull/1) to introduce the filter hook `dokan_dashboard_nav_settings_key`.
- In Dokan WPML (https://github.com/OnTheGoSystems/dokan-wpml/pull/1) to filter `dokan_dashboard_nav_settings_key`.

Note that I created the PRs on our fork for the time being because it's based on other PRs that are still waiting for your review. Once the first PRs are approved and merged, I will change target branches for the subsequent PRs. (I did this otherwise I'll have conflicts and it's more consistent to test all the changes together).
